package.json

```diff
@@ -7,12 +7,12 @@
     "url": "https://github.com/kenchris/urlpattern-polyfill"
   },
   "type": "module",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
```

Comment:
For the record, this shouldn't be needed, but some bundlers (cough Parcel cough) don't support the `exports` map yet.

Reply:
I was just reading iin on all this stuff. Thanks for the PR